project/src/main/java/com/google/sps/data/errors/FetcherException.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.errors;
```

Comment:
I actually think that FetcherException should be in the same package as the Fetcher, as it's a part of its API. 
WDYT?

Reply:
I guess both approaches work. 
In a big server that might be a problem: you might end up having a lot of different exceptions, so it makes more sense to keep everything separated. 
In a smaller server/library this is OK to keep as-is.